test/install.ts

```diff
@@ -290,7 +290,7 @@ test('shrinkwrap compatibility', async function (t) {
   await installPkgs(['rimraf@2.5.1'], testDefaults())
 
   return new Promise((resolve, reject) => {
-    const proc = crossSpawn.spawn('npm', ['shrinkwrap'])
+    const proc = crossSpawn.spawn('npm', ['shrinkwrap'], {stdio: 'inherit'})
```

Comment:
oops, no need in that... that was for debug purposes because the test was failing